### DISCUSSION TITLE
通过metatube刮削出来的图片，分辨率不太高

### DISCUSSION THREAD

**Discussion by byelas** [metatube-community/jellyfin-plugin-metatube#369] [Q&A]
通过metatube刮削出来的图片，包括 封面，背景等所有图片  分辨率不太高 基本上都是800*530，有什么办法可以让分辨率高一些嘛？
另外，还是无法显示演员的照片。我看了日志，没有使用metatube，而是通过sharktube，我安装了本地的gfriends，但是显示错误503 有办法解决嘛？

**Comment by xjasonlyu** [comment]
1. 源站只有800x530分辨率的海报图，没有其他地方找得到高清的图
2. sharktube是什么… 估计是你环境里有其他的东西

**Reply by byelas** [reply]
谢谢大神。
gfriends我刚刚手动搞定了，但是在Jellyfin演员的页面 还是无法点进去。不知道为啥

关于分辨率的问题，是不是所有源都是就800*530的啊 ？除了自己导入高清的之外，还有其他办法嘛？

**Reply by xjasonlyu** [reply]
那你的高清是哪来的我很好奇，难道有什么网站有高清的图么

**Reply by byelas** [reply]
我之前都是在日亚上找的，上面有非常高清的封面图，但是也只有一个封面图。

**Reply by byelas** [reply]
![image](https://github.com/user-attachments/assets/5765954a-9ca4-49bd-84d1-04b1342a62ff)
您看，我metashark（好像是豆瓣的刮削）已经禁用了，并且也没有选择。但是我看jellyfin日志，还是显示在读取，
然后演员的信息无法显示，点进去就是无限转圈。
